includes/product-functions.php

```diff
@@ -441,14 +441,14 @@ function dokan_products_array_filter_editable( $product ) {
  */
 function dokan_product_get_row_action( $post ) {
 
-    if ( empty( $post->ID ) ) {
-        return $row_action;
```

Comment:
This is more preferable condition than your proposed changes, since there is no point of continuing ahead if we do not have `$post->ID`. Meaning, settings `$row_action`, `$row_action_html` and `$product_id` is pointless if `$post->ID` is empty.